packages/order/src/migrations/Migration20240219102530.ts

```diff
@@ -0,0 +1,462 @@
+import { generatePostgresAlterColummnIfExistStatement } from "@medusajs/utils"
+import { Migration } from "@mikro-orm/migrations"
+
+export class Migration20240219102530 extends Migration {
+  async up(): Promise<void> {
+    const sql = `
+      CREATE TABLE IF NOT EXISTS "order_address" (
+          "id" TEXT NOT NULL,
+          "customer_id" TEXT NULL,
+          "company" TEXT NULL,
+          "first_name" TEXT NULL,
+          "last_name" TEXT NULL,
+          "address_1" TEXT NULL,
+          "address_2" TEXT NULL,
+          "city" TEXT NULL,
+          "country_code" TEXT NULL,
+          "province" TEXT NULL,
+          "postal_code" TEXT NULL,
+          "phone" TEXT NULL,
+          "metadata" JSONB NULL,
+          "created_at" TIMESTAMPTZ NOT NULL DEFAULT Now(),
+          "updated_at" TIMESTAMPTZ NOT NULL DEFAULT Now(),
+          CONSTRAINT "order_address_pkey" PRIMARY KEY ("id")
+      );
+
+      CREATE INDEX IF NOT EXISTS "IDX_order_address_customer_id" ON "order_address" (
+          customer_id
+      );
+
+      CREATE TABLE IF NOT EXISTS "order" (
+          "id" TEXT NOT NULL,
```

Comment:
What are some constraints that we can/want to enforce at the DB level on an order?

Reply:
@carlos-r-l-rodrigues I think there are some that are better placed on the db, namely uniqueness. Unless you run all transactions with snapshot isolation there is always a chance that you'll have a race condition that will mess up your data :)

Reply:
I agree. but it's rare the case where we have a unique check that is just a constraint and not an index.